sdk/communication/communication-chat/src/generated/src/models/index.ts

```diff
@@ -401,36 +434,21 @@ export interface ChatThreadSendTypingNotificationOptionalParams
 
 /** Optional parameters. */
 export interface ChatThreadListChatReadReceiptsNextOptionalParams
-  extends coreClient.OperationOptions {
```

Comment:
so are we going to introduce these changes anyways?

Reply:
Generated file so we should not manually modify this